README.md

```diff
@@ -0,0 +1,31 @@
+Titolo
+Breve descrizione progetto
+Tecnologie utilizzate
+Come installare il progetto
+Come far partire il progetto
+Come utilizzare il progetto
+
+# Trip Planner
+
+Hello, this project consists in having an organized schedule of a trip you are planning to undertake. It allows you to filter the contents by type and was created to showcase some of the main functionalities of React.
+
+## Technologies
+
+This project was bootstrapped with [Vite](https://vitejs.dev/) and has been developed using React.
+
+Scss is used as a style preprocessor.
+
+## Installing the project
+
+Run `npm install` in your Terminal to install all the required dependecies.
+
+## Running the project
+
+Run `npm start` in your Terminal. Visit [http://localhost:5173/](http://localhost:5173/) to access the project interface.
+
+## Using Trip Planner
+
+Trip Planner has been created with the purpose of helping you keeping track of your movements when undertaking a trip.
+The App can be used with different types of journey by modifying the `travel_data.json` file with your own details that will be then filtered and categorised by the App.
+By using the filter tool you will be able to see what kind of trasport/ accomodation you are planning to use on each day of your trip - in addition to the events you are planning to partecipate to - without loosing track of many different scheduled departures.

```

Comment:
- `types of entries` invece di `types of journey`
- `src/assets/travel_data.json` (il path completo al file)